packages/css/src/components/label/label.scss

```diff
@@ -15,4 +15,9 @@
 
   @include hyphenation;
   @include text-rendering;
+
+  & > .ams-hint {
+    display: inline-flex;
+    flex-wrap: nowrap;
```

Comment:
Is this the best approach to prevent wrapping? Why not use `white-space: nowrap` which is specifically aimed at handling text?